model_zoo/app_integrate_bundle/ensemble.py

```diff
@@ -119,54 +110,59 @@ def ensemble_inference(self, device, test_datalist, ensemble="Mean"):
         logger.info(f"Total Records in Test Dataset: {len(test_datalist)}")
 
         bundle_inference_config_path = os.path.join(self.bundle_path, "configs", inference_config_paths[0])
-        bundle_inference_config = ConfigParser()
-        bundle_inference_config.read_config(bundle_inference_config_path)
-        bundle_inference_config.update({Const.KEY_BUNDLE_ROOT: self.bundle_path})
-        bundle_inference_config.update({Const.KEY_INFERENCE_DATASET_DATA: test_datalist})
+        inference_workflow = ConfigWorkflow(
+            config_file=bundle_inference_config_path,
+            meta_file=None,
+            logging_file=None,
+            workflow="inference",
+        )
+        inference_workflow.initialize()
 
         # update postprocessing with mean ensemble or vote ensemble
-        post_transform = bundle_inference_config.config["postprocessing"]
-        ensemble_transform = {
-            "_target_": f"{ensemble}Ensembled",
-            "keys": ["pred", "pred", "pred", "pred", "pred"],
-            "output_key": "pred",
-        }
+        raw_postprocessing = inference_workflow.postprocessing.transforms
+        _ensemble_transform = MeanEnsembled if ensemble == "Mean" else VoteEnsembled
+        ensemble_transform = _ensemble_transform(keys=["pred"] * args.n_splits, output_key="pred")
         if ensemble == "Mean":
-            post_transform["transforms"].insert(0, ensemble_transform)
+            _postprocessing = Compose((ensemble_transform, *raw_postprocessing))
         elif ensemble == "Vote":
-            post_transform["transforms"].insert(-1, ensemble_transform)
+            _postprocessing = Compose((*raw_postprocessing, ensemble_transform))
         else:
             raise NotImplementedError
-        bundle_inference_config.update({Const.KEY_INFERENCE_POSTPROCESSING: post_transform})
+
+        inference_workflow.preprocessing = inference_workflow.preprocessing
```

Comment:
Please put 1 line comment to explain why doing this.

Thanks.

Reply:
Have added it in the latest commit, thanks!